### PR TITLE
Fix the accent_color does not exist on type Settings error in tryghost__content-api package

### PR DESCRIPTION
**Pull request by officialrajdeepsingh**
Error I'm fixing the `accent_color` that does not exist on type Settings. In the Settings interface, `accent_color` does not exist, and Now I'm adding a type definition for `accent_color` in the Settings interface.
### Type Error
Screenshot from 2023-04-04 16-14-26

check out the settings API response for more details.

https://ghost.org/docs/content-api/#settings

Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [ ] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: <https://ghost.org/docs/content-api/#settings>
- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

File: types/tryghost__content-api/index.d.ts [hunk 1]
```diff
@@ -59,6 +59,7 @@ export interface Settings extends Metadata, CodeInjection, SocialMedia {
     description?: string | undefined;
     logo?: string | undefined;
     icon?: string | undefined;
+    accent_color?: Nullable<string> | undefined;
```
Comment (sheetalkamat) [inline types/tryghost__content-api/index.d.ts:62]: I am not sure but i think you want this to be `string | undefined` or pls explain why it is different from other properties above.

**Review by sheetalkamat — CHANGES_REQUESTED** [review]

File: types/tryghost__content-api/index.d.ts (hunk 1 above)
Comment (officialrajdeepsingh) [reply types/tryghost__content-api/index.d.ts:62]: Yes, you are right. Firstly, I think `string | undefined` is the right type. But when I [check content-api API docs](https://ghost.org/docs/content-api/#settings) 

The settings also return null.
```bash
GET /content/settings/

```
The `accent_color` colour return null if the value does not exist. Then I check the type definition code. They handle null value with `Nullable<string> | undefined;`. So I used it. With the `Nullable<string> | undefined;` code, I handle null, undefined and string types. I think it is the best way. 

```bash
{
    "settings": {
        "title": "Ghost",
        "description": "The professional publishing platform",
        "logo": "https://docs.ghost.io/content/images/2014/09/Ghost-Transparent-for-DARK-BG.png",
        "icon": "https://docs.ghost.io/content/images/2017/07/favicon.png",
        "accent_color": null,
        "cover_image": "https://docs.ghost.io/content/images/2019/10/publication-cover.png",
        "facebook": "ghost",
        "twitter": "@tryghost",
        "lang": "en",
        "timezone": "Etc/UTC",
        "codeinjection_head": null,
        "codeinjection_foot": "<script src=\"//rum-static.pingdom.net/pa-5d8850cd3a70310008000482.js\" async></script>",
        "navigation": [
            {
                "label": "Home",
                "url": "/"
            },
            {
                "label": "About",
                "url": "/about/"
            },
            {
                "label": "Getting Started",
                "url": "/tag/getting-started/"
            },
            {
                "label": "Try Ghost",
                "url": "https://ghost.org"
            }
        ],
        "secondary_navigation": [],
        "meta_title": null,
        "meta_description": null,
        "og_image": null,
        "og_title": null,
        "og_description": null,
        "twitter_image": null,
        "twitter_title": null,
        "twitter_description": null,
        "members_support_address": "noreply@docs.ghost.io",
        "url": "https://docs.ghost.io/"
    }
}
```

Comment (officialrajdeepsingh) [reply types/tryghost__content-api/index.d.ts:62]: If you think `Nullable<string> | undefined;` is not the right approach. then I replace it with the `string | undefined` type.

Comment (sheetalkamat) [reply types/tryghost__content-api/index.d.ts:62]: Will let  @knguyen0125 @antonve @maveric1977 @thrymgjol determine how to proceed with this.

Comment (officialrajdeepsingh) [reply types/tryghost__content-api/index.d.ts:62]: Hi @sheetalkamat, you are Right `string | undefined;` is the correct type. The docs mention in `https://ghost.org/docs/content-api/#settings` is outdated. I fixed the type and resubmitted the pull request.

**Review by gabritto — APPROVED** [review]

**Pull request closed without merging**